src/modules/lists/lists.service.ts

```diff
@@ -13,4 +14,11 @@ export class ListsService {
     const list = new this.listModel(data);
     return await list.save();
   }
+
+  async findByUserId({_id, public: isPublic }): Promise<List[]> {
```

Comment:
Given that we can use this method to filter with by of the fields (maybe later we can filter by name or something else), let's rename it to just `find`.

```suggestion
async find(filters): Promise<List[]> {
```